docs/cloud-provider-specifics.md

```diff
@@ -31,6 +31,14 @@ aws s3api create-bucket \
     --region <YOUR_REGION> \
     --create-bucket-configuration LocationConstraint=<YOUR_REGION>
 ```
+Note: us-east-1 does not support a `LocationConstraint`.  If your region is `us-east-1`, omit the bucket configuration:
+
+```bash
+
+aws s3api create-bucket \
+    --bucket <YOUR_BUCKET> \
+    --region <YOUR_REGION>
```

Comment:
Sorry for another comment here... but do you think that we should have `us-east-1` here instead of `<YOUR_REGION>`?